sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -66,15 +60,15 @@ add the direct dependency to your project as follows.
 </dependency>
 ```
 [//]: # ({x-version-update-end})
-> Note: This version of the client library defaults to the `"V2021-09-30-preview"` version of the service.
+> Note: This version of the client library defaults to the `"2021-09-30-preview"` version of the service.
 
 This table shows the relationship between SDK versions and supported API versions of the service:
 
 |SDK version|Supported API version of service
 |-|-
 |3.0.x | 2.0
 |3.1.X - 3.1.3| 2.0, 2.1 (default)
-|4.0.0-beta.1 - Latest GA release| V2021-09-30-preview (default)
+|4.0.0-beta.1 - Latest GA release| 2021-09-30-preview (default)
```

Comment:
I like what Python did by adding a clarification of how things are now so different in this section. 
I recommend doing the same.
See https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer#install-the-package